sherpa/astro/ui/tests/test_astro_ui_utils_simulation.py

```diff
@@ -99,10 +100,11 @@ def test_fake_pha_incompatible_rmf(id, clean_astro_ui):
     ebins = np.asarray([1.1, 1.2, 1.4, 1.6, 1.8, 2.0])
     elo = ebins[:-1]
     ehi = ebins[1:]
+    arf = ui.create_arf(elo, ehi)
```

Comment:
The order of checks in `fake_pha` has changed. Now, we need an arf to even get to the point where the error message that is tested here will be thrown.